docs/content/latest/explore/ysql-language-features/triggers.md

```diff
@@ -86,30 +87,28 @@ To start recording changes, you create a function called `record_dept_changes`,
 
 ```sql
 CREATE OR REPLACE FUNCTION record_dept_changes() 
-RETURNS TRIGGER LANGUAGE PLPGSQL
-AS
+RETURNS TRIGGER AS
 $$
-BEGIN
-  IF NEW.department <> OLD.department THEN
-    INSERT INTO employee_dept_changes(employee_no, name, department, changed_on)
-    VALUES(OLD.employee_no, OLD.name, OLD.department, now());
-  END IF;
-
-  RETURN NEW;
+BEGIN 
+IF NEW.department <> OLD.department 
```

Comment:
needs indents all over. and IF..THEN should be on one line
```suggestion
IF NEW.department <> OLD.department THEN
```

Reply:
I've had problems executing this indented.